doc/api/fs.md

```diff
@@ -323,20 +327,25 @@ Stats {
   size: 527,
   blksize: 4096,
   blocks: 8,
+  atimeMs: 1495555358452,
+  mtimeMs: 1495571896565,
+  ctimeMs: 1495571896565,
+  birthtimeMs: 1491689434171,
   atime: Mon, 10 Oct 2011 23:24:11 GMT,
   mtime: Mon, 10 Oct 2011 23:24:11 GMT,
   ctime: Mon, 10 Oct 2011 23:24:11 GMT,
   birthtime: Mon, 10 Oct 2011 23:24:11 GMT }
 ```
 
-Please note that `atime`, `mtime`, `birthtime`, and `ctime` are
-instances of [`Date`][MDN-Date] object and appropriate methods should be used
-to compare the values of these objects. For most general uses
-[`getTime()`][MDN-Date-getTime] will return the number of milliseconds elapsed
-since _1 January 1970 00:00:00 UTC_ and this integer should be sufficient for
-any comparison, however there are additional methods which can be used for
-displaying fuzzy information. More details can be found in the
-[MDN JavaScript Reference][MDN-Date] page.
+*Note*: `atime`, `mtime`, `birthtime`, and `ctime` are instances of
+[`Date`][MDN-Date] object and appropriate methods should be used to compare the
+values of these objects. For most general uses [`getTime()`][MDN-Date-getTime]
```

Comment:
Or maybe this whole paragraph should be rewritten to make note of the `.*Ms` fields first and then simply say that the non-`-Ms` fields are simply `Date`s that use the `.*Ms` values?